### DISCUSSION TITLE
CAGEE with single-cell data?

### DISCUSSION THREAD

**Discussion by ekopania** [hahnlab/CAGEE#30] [Q&A] [answered]
Hello,
I am interested in using CAGEE with single-cell RNAseq data. I was considering using pseudobulk data (i.e., summed counts across cells from the same cell type and sample). Has anyone has tried this approach, or are there are any potential concerns with using pseudobulk instead of bulk RNAseq data with CAGEE? I saw the section in the MBE paper about modeling error across cells within the same cell type, but I am not sure if this would be necessary for pseudobulk data. I'd really appreciate hearing what others think about this. Thank you!
-Emily

**Comment by hahnlab-user** [comment, marked as answer]
Hi Emily,

In theory there is nothing wrong with applying CAGEE to pseudo-bulked data. However, as you probably know, there are lots of sources of error in even calculating the pseudo-bulk values. We are currently developing sc-CAGEE to handle some of this, but there is no code available yet. Feel free to try it out and please tell us what happens!


Matt

**Reply by ekopania** [reply]
Okay, thanks! I'll give it a try and let you know how it goes!
